pillarbox-cast/src/main/java/ch/srgssr/pillarbox/cast/PillarboxCastPlayer.kt

```diff
@@ -617,9 +658,28 @@ class PillarboxCastPlayer internal constructor(
         }
     }
 
+    private inner class CastListener : Cast.Listener() {
+        override fun onVolumeChanged() {
+            Log.d(TAG, "onVolumeChanged")
+            invalidateState()
+        }
+    }
+
     private companion object {
         private const val TAG = "CastSimplePlayer"
-        private val DEVICE_INFO_REMOTE_EMPTY = DeviceInfo.Builder(DeviceInfo.PLAYBACK_TYPE_REMOTE).build()
+
+        /**
+         * @see androidx.media3.cast.CastPlayer.MAX_VOLUME
```

Comment:
This is not yet in the released version of Media3. You can see the explanation here for now:
https://github.com/androidx/media/blob/bd24ec08d13d74f2aae417b85f4cb40092c13f7e/libraries/cast/src/main/java/androidx/media3/cast/CastPlayer.java#L100-L107